tests/test_repositories.py

```diff
@@ -42,6 +44,19 @@
     description="updated",
 )
 
+FEDERATED_REPOSITORY = RemoteRepository(key="test_federated_repository", url="http://test-url.com")

```

Comment:
I think the error you were talking about is because of that line. You wan to create a FEDERATED_REPO but, using thec class `RemoteRepository`

Reply:
this is correct. it's wrong. although it does not cause the mypy errors ;)